autogluon/scheduler/hyperband.py

```diff
@@ -219,6 +219,10 @@ def _run_reporter(self, task, task_job, reporter, searcher, terminator):
         last_updated = None
         while not task_job.done():
             reported_result = reporter.fetch()
+            if 'traceback' in reported_result:
+                logger.exception(reported_result['traceback'])
+                break
```

Comment:
Looking at this, and also FIFOScheduler:
Don't we also need to call
   reporter.move_on()  ?
I am not deep in the details here, but I note this is done if the reporter signals 'done', so should it not also be done if the reporter signals an exception?

Reply:
yep, this is good catch, I added it.